docs/source/examples/managed-jobs.rst

```diff
@@ -183,22 +183,50 @@ Cancel a managed job:
   of the failure. For more details related to provisioning, check :code:`sky jobs logs --controller <job_id>`.
 
 
-Jobs dashboard
-~~~~~~~~~~~~~~
+Viewing jobs in dashboard
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use ``sky jobs dashboard`` to open a dashboard to see all jobs:
+The SkyPilot dashboard, ``sky dashboard`` has a **Jobs** page that shows all managed jobs.
+
+
+.. image:: ../images/dashboard-managed-jobs.png
+  :width: 800
+  :alt: Managed jobs dashboard
+
+The UI shows the same information as the CLI ``sky jobs queue -au``.
+
+
+.. _job-priority:
+
+Job queueing and priority
+-------------------------
+
+Job queueing and priority are supported by managed jobs.
+
+
+Specify job priority by setting the ``job.priority`` field in the job YAML.
+
+.. code-block:: yaml
+
+  job:
+    # Priority of the job, between 0 and 1000 (default: 500).
+    #
+    # A lower value means that the job is higher priority. High priority jobs
+    # are scheduled sooner and will block lower priority jobs from starting
+    # until the high priority jobs have started.
+    priority: 500
 
-.. code-block:: console
 
-  $ sky jobs dashboard
+All jobs are submitted to a queue. When the scheduler is selecting the next job
+to schedule, it will select the highest priority job.
 
-This automatically opens a browser tab to show the dashboard:
+If a high priority job is still launching, lower priority jobs will not be
+scheduled (i.e., stay in pending).
 
-.. image:: ../images/managed-jobs-dashboard.png
+.. image:: https://i.imgur.com/8H8ictY.png
```

Comment:
Why are we using imgur for this?

Reply:
Just following some recent PRs, to save repo size?